testsuite/cluster-test/src/instance.rs

```diff
@@ -4,9 +4,16 @@
 #![forbid(unsafe_code)]
 
 use anyhow::{ensure, format_err, Result};
+use lazy_static::lazy_static;
+use regex::{Error, Regex};
 use serde_json::Value;
 use std::{collections::HashSet, ffi::OsStr, fmt, process::Stdio};
 
+lazy_static! {
```

Comment:
You can consider using lazy from once_cell crate, this is new recommended way in libra(search for Lazy::new, also you can probably seem there is (almost) now lazy_static usage).